cmd/query/app/static_handler_test.go

```diff
@@ -72,7 +73,6 @@ func TestRegisterStaticHandler(t *testing.T) {
 				BasePath:     testCase.basePath,
 				UIConfig:     "fixture/ui-config.json",
 			})
-			assert.Empty(t, buf.String(), "no logs during construction")
```

Comment:
I removed this assertion, as I believe there's value in adding a log message stating that the config file specified via CLI is being watched.